### DISCUSSION TITLE
Updated Districts

### DISCUSSION THREAD

**Discussion by bdavisce21** [openstates/issues#722] [General]
Hello,

Since the legislative districts are being redrawn, is there a time frame on when the new districts will be pulled through the API?

Thank you,
Braden Davis

**Comment by NewAgeAirbender** [comment]
Hello, 

We're working on getting that updated and hoping to have it completed within the next 2 weeks.

Thanks, 
Rylie

**Reply by bdavisce21** [reply]
That is awesome!  Thank you so much for the quick reply!

**Comment by NewAgeAirbender** [comment]
Upon further investigation, due to the fact that there are two valid sets of boundaries, elections & elected, until the end of the year, Open States will continue to provide who is currently in office in each district. Boundaries changed due to redistricting will be updated before 2023 sessions, along with newly elected legislators.

**Reply by cmmelgoza** [reply]
What's the latest on the avails of the 2020 SLDU/SLDL maps?

**Reply by desmond-silveira** [reply]
Might the Open States project reconsider this decision, perhaps by maintaining both sets of data simultaneously and differentiating data with a new decade field?  It would be great to be able to query for both the 2010 district and 2020 district.

**Comment by Tricktronics** [comment]
I see here https://openstates.org/data/geo/ that the latest boundary data support for Texas is 2018. Do you know when the latest boundary data will be available?

**Comment by NewAgeAirbender** [comment]
We are behind our goal in getting the post-redistricting data into the tool. Our usual data source (US Census) has not posted new geographic shapefiles, so we are scrambling to find other sources. If you have any leads, it would be appreciated.

**Reply by Tricktronics** [reply]
Thanks for your quick response. I do not have another source for geo shapefiles. If I find one, I will let you know.
